ui/apps/dashboard/src/components/RunDetails/RunDetails.tsx

```diff
@@ -21,6 +22,15 @@ export function RunDetails({ runID, standalone = true }: Props) {
   const cancelRun = useCancelRun({ envID: env.id, runID });
   const rerun = useRerun({ envID: env.id, envSlug: env.slug, runID });
 
+  const internalPathCreator = useMemo(() => {
```

Comment:
Is there a way to do something similar to this instead of passing a pathCreator, as it would be more reusable for the Dev Server?
https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/components/RunDetails/RunDetails.tsx#L61

Reply:
I don't think so, since we don't know the new run ID until we actually rerun it deep down in the view